<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendMail</class>
 <widget class="QDialog" name="SendMail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enviar e-mail</string>
  </property>
  <property name="locale">
   <locale language="Portuguese" country="Brazil"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelEmail_3">
        <property name="text">
         <string>Servidor SMTP:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditServidor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEmail_2">
        <property name="text">
         <string>Porta servidor:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPorta">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelEmail">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelPasswd">
        <property name="text">
         <string>Senha:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditPasswd">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelDest">
        <property name="text">
         <string>Destinatário:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelCopia">
        <property name="text">
         <string>Cópia:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEditCopia"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelTitulo">
        <property name="text">
         <string>Título:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditTitulo"/>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelAnexo">
        <property name="text">
         <string>Anexos:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEditAnexo"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonBuscar">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelMensagem">
        <property name="text">
         <string>Mensagem:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QTextEdit" name="textEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonEnviar">
           <property name="text">
            <string>Enviar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditEmail"/>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxDest">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditServidor</tabstop>
  <tabstop>lineEditPorta</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditPasswd</tabstop>
  <tabstop>comboBoxDest</tabstop>
  <tabstop>lineEditCopia</tabstop>
  <tabstop>lineEditTitulo</tabstop>
  <tabstop>lineEditAnexo</tabstop>
  <tabstop>pushButtonBuscar</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>pushButtonEnviar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
